tests/Database/DatabaseEloquentCollectionTest.php

```diff
@@ -12,7 +12,7 @@
 
 class DatabaseEloquentCollectionTest extends TestCase
 {
-    protected function tearDown(): void
+    protected function tearDown() : void
```

Comment:
Please revert all these code style changes.

Reply:
Certainly, sorry about that :)